webapp-django/crashstats/supersearch/static/supersearch/js/socorro/search.js

```diff
@@ -67,6 +68,44 @@ $(function () {
                 });
                 $('.tablesorter').tablesorter();
 
+                $('.sort-header', contentElt).each(function() {
+                    $(this).click(function(event) {
```

Comment:
Here's the same code as in the other block of JS. It sets an individual event handler *inside* a loop.